tests/publiccloud/path_and_reboot.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
```

Comment:
you have typo in file name , I suppose you wanted to name a file 'patch_and_reboot.pm'